b0xing/app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,4 @@
 <resources>
-    <string name="app_name">My Application</string>
-
-    <string name="hello_world">Hello world!</string>
+    <string name="app_name">boxr</string>
```

Comment:
I think it looks better if we take out the 0.  Nothing really has numbers as letters anymore :wink: